Dev-Corner/slides/tool_integration.html

```diff
@@ -477,6 +477,83 @@
 
 ###.image-25[![](../images/planemo-logo.png)]
 
+#### `planemo tool_init`
+
+Creates a skeleton of xml file
+
+```bash
+$ mkdir new_tool
+$ cd new_tool
+$ planemo tool_init --id 'some_short_id' --name 'My super tool'
+```
+
+Complicated version:
+
+```bash
+$ planemo tool_init --id 'seqtk_seq' \
```

Comment:
Can you use an other example?
This command solves the half of the exercice of this course 🔨 

What do you think about:
```bash
planemo tool_init \
 --id "samtools_sort" --name "Samtools sort" \
 --description "Sort alignments by leftmost coordinates, or by read nam" \
 --requirement samtools \
 --example_command "samtools sort -o 1.bam null > 1_sorted.bam" \
 --example_input 1.bam \
 --example_output 1_sorted.bam \
 --test_case --version_command "samtools --version | head -1" \
 --help_from_command "samtools sort" \
 --doi 10.1093/bioinformatics/btp352
```